contracts/ChildAccount.cdc

```diff
@@ -140,20 +180,30 @@ pub contract ChildAccount {
             self.grantedCapabilities.insert(key: cap.getType(), cap)
         }
 
+        /// Removes the Capability of given Type from this Tag's grantedCapabilities
+        /// mapping
+        ///
+        /// @param type: The Type of Capability to be removed
+        ///
+        /// @return the removed Capability or nil if it did not exist
+        ///
         access(contract) fun revokeCapability(_ type: Type): Capability? {
             return self.grantedCapabilities.remove(key: type)
         }
 
+        /// Sets the isActive Bool flag to false
+        ///
         access(contract) fun setInactive() {
             self.isActive = false
         }
     }
 
-    /// Wrapper for the child's info and authacct and tag capabilities
+    /// Wrapper for the child's metadata, AuthAccount, and ChildAccountTag Capabilities
     ///
     pub resource ChildAccountController: MetadataViews.Resolver {
```

Comment:
How about ChildAccountOwner as the name, specifically because this is the object that matters because it owns all the important capabilities that make this solution possible.

We also discussed changing ChildAccountManager to ChildAccountAdministrator - being more precise about the role here and less vague than just management